File: backend/models/Banner.js
const { DataTypes } = require("sequelize");
const db = require("../config/db");

const Banner = db.define(
  "Banner",
  {
    id: { type: DataTypes.INTEGER, primaryKey: true, autoIncrement: true },
    tieu_de: { type: DataTypes.STRING },
    hinh_anh_url: { type: DataTypes.STRING },
    duong_dan: { type: DataTypes.STRING },
    vi_tri: { type: DataTypes.STRING },
    thu_tu: { type: DataTypes.INTEGER, defaultValue: 0 },
    ngay_bat_dau: { type: DataTypes.DATE },
    ngay_ket_thuc: { type: DataTypes.DATE },
    trang_thai: { type: DataTypes.STRING, defaultValue: "active" },
  },
  {
    tableName: "Banner",
    timestamps: false,
  },
);

module.exports = Banner;

File: backend-csharp/Models/Banner.cs
using System;
using System.Collections.Generic;

namespace WebFashion.Api.Models;

public partial class Banner
{
    public long Id { get; set; }

    public string? TieuDe { get; set; }

    public string HinhAnhUrl { get; set; } = null!;

    public string? DuongDan { get; set; }

    public string ViTri { get; set; } = null!;

    public int ThuTu { get; set; }

    public DateTime? NgayBatDau { get; set; }

    public DateTime? NgayKetThuc { get; set; }

    public string TrangThai { get; set; } = null!;
}
